sqlagg/base.py

```diff
@@ -320,7 +321,11 @@ def name(self):
 
     @property
     def column_key(self):
-        return self.table_name, str(self.filters), str(self.group_by)
+        return (
+            self.table_name,
+            tuple(sorted(self.filters)) if self.filters else None,
+            tuple(self.group_by) if self.group_by else None
+        )
```

Comment:
This is the main change along with the implementation of correct hashing and comparison for the filter classes